packages/components/stories/config/i18n.js

```diff
@@ -0,0 +1,26 @@
+import i18n from 'i18next';
+import XHR from 'i18next-xhr-backend';
+
+i18n
+	.use(XHR)
```

Comment:
it should work by default without the xhr, we must provide default traduction without traduction. So even bad english will be fixed by the translations :)